rules/validator.go

```diff
@@ -164,11 +169,17 @@ func (v *validator) validateFilters(f map[string]string) error {
 }
 
 func (v *validator) validateRollupTags(ruleName string, tags []string) error {
-	requiredTags := v.opts.RequiredRollupTags()
-	if len(requiredTags) == 0 {
-		return nil
+	// Validate that all tags have valid characters
```

Comment:
The tags passed in here are tag names rather than tag values btw.

Also comment should end w/ a period.